packages/schematics/angular/config/schema.json

```diff
@@ -16,13 +16,36 @@
     "type": {
       "type": "string",
       "description": "Specifies the type of configuration file to generate.",
-      "enum": ["karma", "browserslist"],
+      "enum": ["karma", "browserslist", "ai"],
       "x-prompt": "Which type of configuration file would you like to create?",
       "$default": {
         "$source": "argv",
         "index": 0
       }
+    },
+    "tool": {
```

Comment:
Should this be a multi-select prompt?
We could still default to having them all selected. Would need to change the type to a string array though.  But it's probably unlikely that everyone would want all available tools when not specifying the option.  Especially since the option may not be known to even exist.

Reply:
As mentionned by @alan-agius4 [here](https://github.com/angular/angular-cli/pull/30763#pullrequestreview-3046403497), the prompt would show for the others options. Or do you have an alternative in mind ?